### PR TITLE
LP-2630 - Add mandrill support for multiple emails

### PR DESCRIPTION
**Pull request by MunamMubashir-arbisoft**
[LP-2630](https://philanthropyu.atlassian.net/browse/LP-2630)

- Add mandrill support for a group of email addresses to send the email to.
- Change all the occurrences of the send email task to conform with the new changes.

### CODE REVIEW

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1440/workflows/workflows_id/jobs/1440/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/common/lib/mandrill_client/client.py (33.3%): Missing lines 71-72
openedx/adg/common/lib/mandrill_client/email_data.py (100%)
openedx/adg/common/lib/mandrill_client/tasks.py (90.9%): Missing lines 34
openedx/adg/common/lib/mandrill_client/tests/test_tasks.py (100%)
openedx/adg/lms/applications/helpers.py (100%)
openedx/adg/lms/student/helpers.py (100%)
-------------
Total:   25 lines
Missing: 3 lines
Coverage: 88%
-------------
```

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1442/workflows/workflows_id/jobs/1442/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/common/lib/mandrill_client/client.py (33.3%): Missing lines 71-72
openedx/adg/common/lib/mandrill_client/email_data.py (100%)
openedx/adg/common/lib/mandrill_client/tasks.py (100%)
openedx/adg/common/lib/mandrill_client/tests/test_tasks.py (100%)
openedx/adg/lms/applications/helpers.py (100%)
openedx/adg/lms/student/helpers.py (100%)
-------------
Total:   24 lines
Missing: 2 lines
Coverage: 91%
-------------
```

**Review by zahid-arbisoft — CHANGES_REQUESTED** [review]

File: openedx/adg/common/lib/mandrill_client/tests/test_tasks.py [hunk 1]
```diff
@@ -3,11 +3,14 @@
...
 from openedx.adg.common.lib.mandrill_client.tasks import task_send_mandrill_email
 
 
+@pytest.mark.django_db
 @patch('openedx.adg.common.lib.mandrill_client.tasks.MandrillClient')
 def test_task_send_mandrill_email(mock_mandrill_client):
-    task_send_mandrill_email('email_template', 'user@email.com', {'content': 'content'})
+    task_send_mandrill_email('email_template', ['user@email.com'], {'content': 'content'})
 
     mock_mandrill_client.assert_called_once()
```
[the remark this answers is not in the document]
Comment (MunamMubashir-arbisoft) [reply openedx/adg/common/lib/mandrill_client/tests/test_tasks.py:16]: Actually this was a single point ticket that blocked a few others so we wanted to get this done sooner, therefore I didn't really go through the unit tests properly.
However, this is a good suggestion. And I'll cater it soon

[the remark this answers is not in the document]
Comment (zahid-arbisoft) [reply openedx/adg/common/lib/mandrill_client/tests/test_tasks.py:16]: In that case, I can approve PR, you can create a follow-up PR or separate story for the unit test. WDYT

[the remark this answers is not in the document]
Comment (MunamMubashir-arbisoft) [reply openedx/adg/common/lib/mandrill_client/tests/test_tasks.py:16]: I'll add the test, no worries.

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1443/workflows/workflows_id/jobs/1443/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/common/lib/mandrill_client/client.py (33.3%): Missing lines 71-72
openedx/adg/common/lib/mandrill_client/email_data.py (100%)
openedx/adg/common/lib/mandrill_client/tasks.py (100%)
openedx/adg/common/lib/mandrill_client/tests/test_tasks.py (100%)
openedx/adg/lms/applications/helpers.py (100%)
openedx/adg/lms/student/helpers.py (100%)
-------------
Total:   26 lines
Missing: 2 lines
Coverage: 92%
-------------
```

**Review by zahid-arbisoft — APPROVED** [review]
LGTM, nice work 🚀

**Review by asadali145 — APPROVED** [review]

**Pull request merged**